std/src/main/scala/algebra/std/map.scala

```diff
@@ -26,33 +50,80 @@ class MapEq[K, V](implicit ev: Eq[V], a: AdditiveMonoid[V]) extends Eq[Map[K, V]
     check(x, y) && check(y, x)
```

Comment:
:+1: We took that approach in Spire. The default was the strict version, but you could `import spire.optional.vectorEq` (or somethign similar) that would do "vector" eq/order (which was required for the law tests to pass). Of course, an alternative could be to wrap the maps in a value class or something for the vector-style operations, but that comes with another host of usability issues.